packages/Vendor/src/ValueObjects/VendorDisplayName.php

```
@@ -0,0 +1,36 @@
+namespace Nexus\Vendor\ValueObjects;
+
+final readonly class VendorDisplayName
+{
+    private string $value;
+
+    public function __construct(string $value)
+    {
+        $normalized = trim($value);
+
+        if ($normalized === '') {
+            throw new \InvalidArgumentException('Vendor display name cannot be empty.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '16', column '23'). (undefined)

(MissingImport)